.github/workflows/release.yml

```
@@ -163,6 +177,7 @@ jobs:
           # authored by a real user identity and CI triggers on them — see
           # the comment on the Checkout step for the full rationale.
           GITHUB_TOKEN: ${{ secrets.RELEASE_PAT }}
+          SB_VERSION: ${{ steps.release-gate.outputs.sb_version }}
```

actionlint (1.7.12)

[error] property "release-gate" is not defined in object type {crates-auth: {conclusion: string; outcome: string; outputs: {string => string}}; fixtures-cache: {conclusion: string; outcome: string; outputs: {string => string}}; pre: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)